pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Related PRs
+List related PRs against other branches:
+
+branch | PR
+------ | ------
+other_pr_production | [link]()
+other_pr_master | [link]()
+
+
+## Todos
+- [ ] Tests
+- [ ] Documentation
+
+
+## Deploy Notes
```

Comment:
Deploy notes, if any, makes sense for an application, but in our case, once the code is merged, it will get versioned and deployed via `npm publish`. There are the demo websites, but I imagine they automatically deploy to Netlify once merged to the main branch @raae?

Reply:
Yes, that is correct (or at least very soon will be) @nickytonline. So I agree.